### ISSUE TITLE
Concerning documentation of Species

### ISSUE THREAD

**Issue by moralesq** [lamyj/sycomore#4]
Thanks for sharing this work! 

I have a question regarding how to specify dephasing in Specifies. In the documentation dephasing is set with `delta_omega=0*rad/s.` The use of rad in the units suggests this is an angular frequency, i.e., `w = 2pi*f`. However, in the code it is treated as a frequency:

```
auto const angular_frequency =
        2*M_PI * (
            // Field-related dephasing
            delta_omega_Hz
            // FIXME: delta_omega of species 
            // // Species-related dephasing, e.g. chemical shift or susceptibility
            // + species.delta_omega
        )
```
 
 I'm I missing something, or should the documentation be changed to delta_omega=0*Hz?
 
 Thanks!

**Comment by lamyj** [comment]
You're right, there is an inconsistency in the units. Note that the code you are referring to ([sycomore::isochromat::Model::build_time_interval](https://github.com/lamyj/sycomore/blob/master/src/sycomore/isochromat/Model.cpp#L119-L128)) is still in its testing stage, and that the whole off-resonance part of the simulations (both $\Delta B_0$ and $R_2^*$) is being reworked. I'll keep this issue open while the rework is ongoing so that you can be notified when things are stable.

**Comment by lamyj** [comment]
This took quite a bit longer than expected, but this should be solved with the just-released v2.0.0. Don't hesitate to re-open if needed.
